sdk/eventhub/Azure.Messaging.EventHubs/samples/Azure.Messaging.EventHubs.Samples.csproj

```diff
@@ -11,7 +11,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Data.SchemaRegistry" VersionOverride="1.0.0-beta.2" />
```

Comment:
nit:  Can we add comments that the version overrides are temporary until these hit GA?

Reply:
Comments have been added.